k8s/leader-election-leases.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: logdna-agent-reporter-lease
+  namespace: logdna-agent
+rules:
+- apiGroups: ["coordination.k8s.io"]
+  resources: ["leases"]
+  verbs: ["get", "watch", "list", "update", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: logdna-agent-reporter-lease
+  namespace: logdna-agent
+subjects:
+- kind: ServiceAccount
+  name: logdna-agent
+  namespace: logdna-agent
+roleRef:
+  kind: Role
+  name: logdna-agent-reporter-lease
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: coordination.k8s.io/v1
+kind: LeaseList
```

Comment:
Is there a need to use LeaseList here? I looks like there's only one Lease that's needed for this feature?

Reply:
fixed